packages/manager/apps/pci-private-network/src/pages/new/new.constants.ts

```diff
@@ -0,0 +1,41 @@
+import { z } from 'zod';
+
+export const VLAN_ID = {
+  default: 1,
+  min: 0,
+  max: 4000,
+};
+
+export const DEFAULT_CIDR = '10.{vlanId}.0.0/16';
+
+export const GATEWAY_HOURLY_PLAN_CODE = 'gateway.s.hour.consumption';
+
+export const schema = z.object({
```

Comment:
It's an const exported. I think it can have same name struct. And maybe more readable.

```suggestion
export const NEW_PRIVATE_NETWORK_FORM_SCHEMA = z.object({
```